### ISSUE TITLE
[Bug] SEO entry previews displays wrong data in multi-site multi-domain configuration

### ISSUE THREAD

**Issue by naboo** [nystudio107/craft-seomatic#324]
So when using SEOmatic in a multi-site multi-domain setup and you set the `baseCpUrl` to the license domain the SEO entry previews doesn't show the correct data. In my example it shows the URL defined in `baseCpUrl` even though I'm viewing an entry from "Site B" which has a completly different domain.

Exmaple here:
![skarmklipp 1](https://user-images.githubusercontent.com/1220045/52959300-29265600-3396-11e9-957f-61ebbbd3b2fa.PNG)

The same issue seems to also effect the Content SEO section of SEOmatic in which it doesn't display the correct entry count. This site has at least ~50 entries in various sections but they all say "0". Example:
![screenshot_2019-02-18 seomatic - content seo - nobis](https://user-images.githubusercontent.com/1220045/52959361-4b1fd880-3396-11e9-9965-9530f4a3c3ff.png)

### How to reproduce
1. Setup Craft with two different sites using two different domains (not sharing same top-domain)
2. Set the `baseCpUrl` to the domain of Site A ("domain holding license")
3. Check the SEO previews of entry in Site B

### Versions
Craft: 3.1.12
SEOmatic: 3.1.43

**Comment by khalwat** [comment]
Any idea why the `{{ seomatic.meta.seoDescription }}` isn't parsing?

**Comment by khalwat** [comment]
This might be a case where having access to the site would be really helpful in diagnosing things... if that's possible @naboo ?

**Comment by naboo** [comment]
In this case the site - which is the `baseCpUrl` - just doesn't have any entries and/or content. It basically only has its "Homepage" single and nothing else. So maybe SEOmatic just can't fetch any meta to parse (if its trying to pull it from the base URL)?.

I'll try and catch you on Discord later today with some credentials 👍

**Comment by khalwat** [comment]
Is this still an issue for you @naboo ?

**Comment by naboo** [comment]
Yeah so I solved one issue, which is the lack of the meta description preview. When I toggled the field in Content SEO settings it had the value `{{ seomatic.meta.seoDescription }}` which is what it says in the preview. I just cleared it, resaved, and the preview started working.

What I haven't figured out it why the `baseCpUrl` always is shown under (for example) the Facebook preview BUT the actual preview URL (like the link applied to the preview image) is correct with the current site you are viewing.

So it's _kinda_ solved :P
